File: src/main/java/no/nav/arbeidsgiver/sykefravarsstatistikk/api/statistikk/sykefraværshistorikk/LegemeldtSykefraværsprosent.java
package no.nav.arbeidsgiver.sykefravarsstatistikk.api.statistikk.sykefraværshistorikk;

import lombok.AllArgsConstructor;
import lombok.Getter;
import no.nav.arbeidsgiver.sykefravarsstatistikk.api.statistikk.Statistikkategori;

import java.math.BigDecimal;

@AllArgsConstructor
@Getter
public class LegemeldtSykefraværsprosent {
    private final Statistikkategori type;
    private final String label;
    private final BigDecimal prosent;
}

File: src/main/java/no/nav/arbeidsgiver/sykefravarsstatistikk/api/statistikk/sykefraværshistorikk/LegemeldtSykefraværsprosent.kt
package no.nav.arbeidsgiver.sykefravarsstatistikk.api.statistikk.sykefraværshistorikk

import no.nav.arbeidsgiver.sykefravarsstatistikk.api.statistikk.Statistikkategori
import java.math.BigDecimal

data class LegemeldtSykefraværsprosent(
    val type: Statistikkategori? = null,
    val label: String? = null,
    val prosent: BigDecimal? = null
)